lib/modules/apostrophe-assets/public/css/vendor/jquery-ui-3.less

```diff
@@ -49,13 +48,13 @@
 	left: 0;
 	position: absolute;
 	opacity: 0;
-	filter:Alpha(Opacity=0); /* support: IE8 */
 }
 
 .ui-front {
-  z-index: @apos-z-index-max;
+	z-index: @apos-z-index-max;
 }
 
+
```

Comment:
changes in this files seem to be related to dropping IE

Reply:
Yeah good it'll make it slightly lighter